pkgs/development/compilers/llvm/10/llvm/default.nix

```
@@ -244,6 +245,12 @@ in stdenv.mkDerivation (rec {
     ln -s $lib/lib/libLLVM.dylib $lib/lib/libLLVM-${shortVersion}.dylib
     ln -s $lib/lib/libLLVM.dylib $lib/lib/libLLVM-${release_version}.dylib
   ''
+  + optionalString (enableSharedLibraries) ''
+    mkdir -p $dev/lib
+    mv $lib/lib/*.a $dev/lib
+    ${gnused}/bin/sed -i -E "s|$lib/lib/(.*)\.a|$dev/lib/\1\.a|" \
```

```suggestion
    sed -i -E "s|$lib/lib/(.*)\.a|$dev/lib/\1\.a|" \
```
part of the stdenv if I am not mistaken.

---

Ah damn, I wasn't sure. Simplified accordingly, thanks.